src/dotnet/commands/dotnet-run/RunCommandParser.cs

```diff
@@ -51,7 +51,8 @@ internal static class RunCommandParser
                         "--no-build",
                         LocalizableStrings.CommandOptionNoBuildDescription,
                         Accept.NoArguments()),
-                    CommonOptions.NoRestoreOption()
+                    CommonOptions.NoRestoreOption(),
+                    CommonOptions.VerbosityOption()
```

Comment:
There is a bug out for setting the verbosity for run. I will find and reference it here. We can use this PR to close both issues.

Reply:
I'll amend the commit message with a reference.

Reply:
Is it #4823 (partial fix here) or another issue?

Reply:
https://github.com/dotnet/cli/issues/7932

Reply:
Pushed in ea85555.